lesson08/assignment/video_cli/l8/src/App.vue

```diff
@@ -3,19 +3,17 @@
     <Navbar />
     <router-view />
     <ContactForm />
-    <post
-    v-for="post in posts"
-    v-bind:post="post"
-    v-bind:key="post.id">
+    <post v-for="(post, id) in posts"
```

Comment:
This was rewritten to generate an auto-incrementing id each time through the loop, so you don't need to create a unique key in your data.